src/main/java/frc/team2412/robot/Controls.java

```diff
@@ -105,6 +107,9 @@ public void bindDriveControls() {
                 driveController.getLeftYAxis(), driveController.getLeftXAxis(),
                 driveController.getRightXAxis()));
         resetDriveGyroButton.whenPressed(() -> subsystems.drivebaseSubsystem.resetGyroAngle(Rotation2.ZERO));
+
+        if (Robot.getTypeFromAddress() == RobotType.DRIVEBASE)
+            setPoseButton.whenPressed(() -> subsystems.drivebaseSubsystem.setPose());
```

Comment:
Add Robot.isDriveBase() instead

Reply:
Then revert the changes to make things public

Reply:
Switched to using isCompetition